content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -183,15 +183,47 @@ kfctl apply -V -f ${CONFIG_URI}
 * **${KF_DIR}** - The full path to your Kubeflow application directory.
 
 ### Multi-user, auth-enabled
-Run the following commands to deploy Kubeflow with GitHub OAuth application as authentication provider by dex. To support multi-users with authentication enabled, this guide uses [dex](https://github.com/dexidp/dex) with [GitHub OAuth](https://developer.github.com/apps/building-oauth-apps/). Before continue, refer to the guide [Creating an OAuth App](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/) for steps to create an OAuth app on GitHub.com.
 
-The scenario is a GitHub organization owner can authorize its organization members to access a deployed kubeflow. A member of this GitHub organization will be redirected to a page to grant access to the GitHub profile by Kubeflow.
-
-1. Create a new OAuth app in GitHub. Use following setting to register the application:
-    * Homepage URL: `http://localhost:8080/`
-    * Authorization callback URL: `http://localhost:8080/dex/callback`
-1. Once the application is registered, copy and save the `Client ID` and `Client Secret` for use later.
-1. Setup environment variables:
+Run the following steps to deploy Kubeflow with [IBM Cloud AppID](https://cloud.ibm.com/catalog/services/app-id)
+as authentication provider. 
+
+The scenario is a Kubeflow cluster admin configures Kubeflow as an web
+application in AppID and manages user authentication with builtin
+identity providers (Cloud Directory, SAML, etc) or custom providers.
+
+1. Follow the guide [Getting started with App ID](https://cloud.ibm.com/docs/appid?topic=appid-getting-started)
+to create an AppID service instance.
+2. Follow the step [Registering your app](https://cloud.ibm.com/docs/appid?topic=appid-app#app-register)
```

Comment:
2. Follow the step [Registering your app](https://cloud.ibm.com/docs/appid?topic=appid-app#app-register)
to create an application with type _regularwebapp_ under the provisioned AppID